src/React.Router/SetServerResponse.cs

```diff
@@ -36,7 +36,7 @@ public static class SetServerResponse
 		/// <param name="Response">The response object to use.</param>
 		public static void ModifyResponse(RoutingContext context, HttpResponse Response)
 		{
-			var statusCode = context.status.Value;
+			var statusCode = context.status ?? 302;
```

Comment:
Why is this change needed?

Reply:
If a status code is omitted we can't know if the author intended a permanent or temporary redirect, therefore it's safest to assume temporary